lib/camt_parser/053/statement.rb

```diff
@@ -39,7 +39,7 @@ def electronic_sequence_number
 
       def opening_balance
         @opening_balance ||= begin
-          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "PRCD")]').first.ancestors('Bal')
+          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "OPBD") or contains(text(), "PRCD")]').first.ancestors('Bal')
```

Comment:
I am just wondering because I could not find PRCD in the documentation you linked but I know we have it in the camt files we receive from one of the banks we are working with.
This is also what the existing example files are based on.
You are right though that there is only one child element for it. Alphabetical order is fine then.